TheatricalMovieTrailers/Model/MovieInfoDataStore.swift

```diff
@@ -7,10 +7,22 @@
 
 import Combine
 import Network
-import UIKit.UIImage
+#if os(iOS)
+import UIKit
+#elseif os(macOS)
+import AppKit
+#endif
 import TelemetryClient
 
 class MovieInfoDataStore: ObservableObject {
+    #if os(iOS)
+    typealias Image = UIKit.UIImage
+    let appGroupID = "group.cafe.chrisp.tmt"
+    #elseif os(macOS)
+    typealias Image = AppKit.NSImage
+    let appGroupID = "cafe.chrisp.tmt-group"
+    #endif
```

Comment:
SwiftUI already gives you an `Image` abstraction. I'd strongly recommend using that instead of `UIImage`/`NSImage`.

Reply:
This is a great tip. I had gone down the "path of least resistance" when writing the Mac app and just updated the existing code to use `NSImage` where needed. Now I've re-written it to use `SwiftUI.Image` where applicable and it's much cleaner.